apps/console/src/features/applications/components/settings/attribute-management/attribute-list-item.tsx

```diff
@@ -62,9 +62,9 @@ interface AttributeListItemPropInterface extends TestableComponentInterface {
 /**
  * Selected Attribute list item component.
  *
- * @param {AttributeListItemPropInterface} props - Props injected to the component.
+ * @param props - Props injected to the component.
  *
- * @return {React.ReactElement}
+ * @returns
```

Comment:
Can we mention what we are returning?

Reply:
resolved by https://github.com/wso2/identity-apps/pull/3522/commits/a643633e78865647ca46a73007235365cd6ababf